src/widgets/_index.scss

```
@@ -69,6 +69,15 @@ selection {
     border-radius: 9999px;
 }
 
+.avatar {
+    border: 1px solid #{'alpha(black, 0.35)'};
+    border-radius: 50%;
+    box-shadow:
+        inset 0 0 0 1px #{'alpha(white, 0.05)'},
+        inset 0 1px 0 0 #{'alpha(white, 0.45)'},
+        inset 0 -1px 0 0 #{'alpha(white, 0.15)'};
```

This should probably use `outset-highlight` instead and this is also missing the shadow

```suggestion
    box-shadow:
        outset-highlight(),
        outset-shadow(3);
```